src/__stories__/data-card-story.tsx

```diff
@@ -35,7 +35,7 @@ export default {
 
```

Comment:
Data Card and Media card have new space between Tag and Headline. (+8px, 16px in total)

Reply:
done